Makefile

```diff
@@ -618,4 +441,4 @@ clean: ## Clean the build and test artifacts
 
 ##@ Test Dev Cluster with and without topology
 include test-cluster-without-topology.mk
```

Comment:
test-cluster-without-topology.mk also needs updating to use the clusterctl binary directly